backend/de.metas.ui.web.base/src/main/java/de/metas/ui/web/window/controller/Execution.java

```diff
@@ -221,6 +245,7 @@ private ExecutionBuilder()
 						}
 					}
 
+					assert versionException != null;
```

Comment:
Asserts are pretty unusual in our code and i think they are not enabled when we run on the server.
To make is less surprising i suggest you use Check.assume

Reply:
@teosarca

Reply:
@metas-rc  pls change it
@metas-ts  let's discuss some time